README.md

```diff
@@ -124,19 +130,28 @@ For reference, here is a exp of a step-by-step evaluation:
 6 + 4
 10
 ```
-
+// 3*2
+// 2 + 1
+// = 9
 ## Problem Seven
 
 Write a step-by-step evaluation for the following expression (remember order of operations): `4 / 2 + 8 / 4`.
-
+// 4/2 
+// 8/4
+// 2+2
+// = 4
 ## Problem Eight
 
 Write a step-by-step evaluation for the following expression: `'ca' + 'ter' + 'pi' + 'llar'`.
+// 'cater' + 'pillar'
 
 ## Problem Nine
 
 Write a step-by-step evaluation for the following expression: `2 * 4 === 8 && 'car' + 'pool' === 'carpool'`.
-
+true
+carpool
 ## Problem Ten
 
 Write a step-by-step evaluation for the following expression: `'1' + '2' + '3' - '1'`.
+12 + 2
+= 14
```

Comment:
Not exactly. It keeps building from left to right. should be "123" - "1" which will result in "122". Please make sure to always check your work by running your code. This step is required.